src/btree/bt_sync.c

```diff
@@ -182,8 +182,11 @@ __sync_page_skip(
      * FIXME: Read internal pages from non-logged tables when the remove/truncate
```

Comment:
Can this FIXME comment go away? I think maybe 'no' because this change only affects non-timestamped, logged tables. But I'm not really sure.

Reply:
I'd suggest modifying the large comment block here to say "Skip checkpoint cleanup and reading of internal pages if ..."

Reply:
I can review and revise this block. I will need to find out who to check with as to what "fast delete" refers to exactly.

> Skip checkpoint cleanup and reading of internal pages if ...

As this is function only processes a single page I think this comment would be better suited to a location further up in the call chain.